src/main/webapp/app/overview/course-conversations/dialogs/channels-overview-dialog/channels-overview-dialog.component.ts

```diff
@@ -136,7 +139,7 @@ export class ChannelsOverviewDialogComponent extends AbstractDialogComponent imp
             )
             .subscribe({
```

Comment:
There should be a new line between every funktion, not sure why the client style checks the the one above pass.

Reply:
Why should there be an empty line here? This is a chained function call.

Reply:
oh ok, I added a new line.